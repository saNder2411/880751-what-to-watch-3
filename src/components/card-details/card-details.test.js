import React from 'react';
import renderer from 'react-test-renderer';
import CardDetails from './card-details.jsx';

const data = {
  director: `Steven Spielberg`,
  starring: [`Judi Dench`, `Robert De Niro`, `Leonardo DiCaprio`],
  runtime: `1h 58m`,
  genre: `Drama`,
  released: 1978,
};

it(`Should CardDetails render correctly`, () => {
  const markup = renderer
    .create(<CardDetails {...data} />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
