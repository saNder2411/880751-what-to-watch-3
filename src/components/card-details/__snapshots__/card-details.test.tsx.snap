// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CardDetails render correctly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Steven Spielberg
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Judi Dench
         
        <br />
        Robert De Niro
         
        <br />
        Leonardo DiCaprio
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        2h 26m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Drama
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        1978
      </span>
    </p>
  </div>
</div>
`;
